edxlearndot/models.py

```diff
@@ -19,14 +19,15 @@ class Meta:
         unique_together = ("learndot_component_id", "edx_course_key")
 
     def __str__(self):
-        return self.__unicode__().encode("utf8")
+        return str(self.__unicode__())
```

Comment:
Thanks for finding this @eLRuLL !
Could you add a unit test for this, and make sure the  [`tox` tests run as documented|https://github.com/open-craft/edx-learndot#running-edx-learndot-tests]? I tried briefly to run them but got some dependency issues, so there might be some requirements changes.

Reply:
yep for sure, added :+1: